usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -3836,7 +3906,8 @@ static CK_RV dh_generate_keypair(STDLL_TokData_t * tokdata,
     dh_pgs.pg = malloc(p_len * 2);
     if (!dh_pgs.pg) {
         TRACE_ERROR("%s Memory allocation failed\n", __func__);
-        return CKR_HOST_MEMORY;
+        rc = CKR_HOST_MEMORY;
+        goto dh_generate_keypair_end;
```

Comment:
detected bugfix sneaking in ; -)
please make it a seperate commit even if its just 2 locs

Reply:
will do